main.tf

```diff
@@ -131,9 +131,9 @@ resource "random_password" "rds_db_admin_password" {
   }
 }
 
-data "aws_kms_alias" "rds" {
-  name = "alias/aws/rds"
-}
+# data "aws_kms_alias" "rds" {
+#   name = "alias/aws/rds"
```

Comment:
Please remove it , if its not required.

Reply:
If we remove the resurce of aurora kms key and alias then this would be used